src/main/java/net/minecraftforge/event/CommandEventPost.java

```diff
@@ -0,0 +1,43 @@
+package net.minecraftforge.event;
+
+import net.minecraft.command.ICommand;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.command.PlayerSelector;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+/**
+ * CommandEventPost is fired <b>after</b> a command has been executed.
+ * 
+ * @see CommandEvent
+ */
+public class CommandEventPost extends Event
```

Comment:
Why not make this an inner class of CommandEvent? That seems to be the Forge convention...